team.html

```diff
@@ -232,7 +232,10 @@ <h2 class="text-center">
             <div class="clearfix"></div>
             <br>
             <br>
-            <h3>Core Volunteers</h3>
+            <h3>Core Volunteers & Staff Information</h3>
```

Comment:
Let make it just Staff information instead of Core Volunteers & Staff Informaiton